app/mixins/setup-user-role.js

```diff
@@ -3,6 +3,7 @@ export default Ember.Mixin.create({
   setupUserRole() {
     let session = this.get('session');
     let userRole = session.get('data.authenticated.role');
+    session.set('data.authenticated.userRole', userRole);
```

Comment:
Why not just pull the user's role from data.authenticated.role?  Why double store it?